### PR TITLE
fix: density slider and color pickers in component explorer

### PR DESCRIPTION
**Pull request by eljefe223**
# Description
There was a few bugs related to design system changes in the component explorer. This cleans up how this works and bridges communication between the sliders and pickers at the bottom of the component explorer and the design system config panel.

Closes: #1783 

## Motivation & context

1. Changing color string in design system config panel crashed the component.
2. Changing the background color picker was not reflected in the design system config panel
3. The density slider no longer worked and was not reflected in the design system config panel.
4. The accent color picker no longer worked.

## Issue type checklist

- [ ] **Chore**: A change that does not impact distributed packages.
- [X] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

## Process & policy checklist

- [ ] I have added tests for my changes.
- [X] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [X] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://microsoft.github.io/fast-dna/docs/en/contributing/standards) for this project.

### CODE REVIEW

File: packages/fast-components-styles-msft/src/design-system/design-system.schema.json [hunk 1]
```diff
@@ -8,7 +8,8 @@
         "backgroundColor": {
             "title": "Background color",
             "type": "string",
-            "default": "#FFFFFF"
+            "default": "#FFFFFF",
+            "disabled": "true"
```
Comment (janechu) [inline packages/fast-components-styles-msft/src/design-system/design-system.schema.json]: boolean should not be in quotes

File: packages/fast-components-react-msft/app/app.tsx [hunk 2]
```diff
@@ -200,45 +189,64 @@ export default class App extends React.Component<{}, AppState> {
...
+    }
+
     private handleUpdateTheme = (theme: ThemeName): void => {
         if (theme !== ThemeName.custom) {
             this.setState({
                 theme,
                 backgroundColor: theme === ThemeName.dark ? dark : light,
+                designSystem: clone(
+                    merge({}, this.state.designSystem, {
+                        backgroundColor: theme === ThemeName.dark ? dark : light,
```
Comment (nicholasrice) [inline packages/fast-components-react-msft/app/app.tsx]: what are `dark` and `light`? I think this is where we would want to pull the background value from the current design-system

File: packages/fast-components-react-msft/app/app.tsx [hunk 3]
```diff
@@ -281,7 +291,11 @@ export default class App extends React.Component<{}, AppState> {
 
     private handleDensityUpdate = (e: React.ChangeEvent<HTMLInputElement>): void => {
         this.setState({
-            density: parseInt(e.target.value, 10) as DensityOffset,
+            designSystem: clone(
```
Comment (nicholasrice) [inline packages/fast-components-react-msft/app/app.tsx]: Why do you need to `clone` _and_ `merge`?

File: packages/fast-components-react-msft/app/app.tsx (hunk 2 above)
Comment (nicholasrice) [reply packages/fast-components-react-msft/app/app.tsx]: nm - this should be a different PR

**Pull request merged**